components/movementsensor/adxl345/adxl345.go

```diff
@@ -232,6 +331,123 @@ func (adxl *adxl345) writeByte(ctx context.Context, register, value byte) error
 	return handle.WriteByteData(ctx, register, value)
 }
 
+func getByteFromInterrupts(interrupts []string) byte {
+	var register byte
+	for _, interrupt := range interrupts {
+		register += interruptBitPosition[interrupt]
+	}
+	return register
+}
+
+func (adxl *adxl345) enableInterrupts(ctx context.Context) error {
+	register := getByteFromInterrupts(adxl.interruptsEnabled)
+	return adxl.writeByte(ctx, IntEnable, register)
+}
+
+func (adxl *adxl345) setRelevantRegisters(ctx context.Context) error {
+	if len(adxl.interruptsEnabled) == 0 {
+		return nil
+	}
+	relevantRegisters := adxl.getRelevantRegisterValues()
+	for key := range defaultRegisterValues {
+		val, ok := relevantRegisters[key]
+		if ok {
+			err := adxl.writeByte(ctx, key, val)
+			if err != nil {
+				return errors.Errorf("Unable to write byte at register %x", key)
+			}
+		} else {
+			err := adxl.writeByte(ctx, key, defaultRegisterValues[key])
+			if err != nil {
+				return errors.Errorf("Unable to write byte at register %x", key)
+			}
+		}
+	}
+	return nil
+}
+
+func (adxl *adxl345) getAxes(x, y, z bool) byte {
+	var tapAxes byte
+	if x {
+		tapAxes += xBit
+	}
+	if y {
+		tapAxes += yBit
+	}
+	if z {
+		tapAxes += zBit
+	}
+	return tapAxes
+}
+
+func (adxl *adxl345) getRelevantRegisterValues() map[byte]byte {
```

Comment:
At first glance, I don't see what this function is doing. Put in a 1-line comment at the top. "This returns a map from register addresses to data which should be written to that register to configure the interrupt pin" or something like that, I think?